src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -309,6 +309,17 @@ Below is a list of available sub-commands:
 | `--verbose`                      | Produce verbose logs during validation
 <!-- markdownlint-enable line-length -->
 
+{{% notice warning %}}
+In Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
+warnings in subctl's output. subctl should work fine, but you can avoid the warnings and ensure correct behavior by adding the below labels 
```

Comment:
```suggestion
warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by adding these labels 
```

The binary `subctl` should be `code style` and ideally shouldn't' start a sentence. Please see the Docs Style Guide for details. https://submariner.io/development/website/style-guide/